modules/AWS/eks/iam.tf

```diff
@@ -0,0 +1,37 @@
+resource "aws_iam_role" "node_group" {
```

Comment:
Create iam role and attach EKS worker node policies to it + osquery policies (if enabled). Due to a bug, it's not possible to use `iam_role_additional_policies` and let the module automatically create iam role. Such policies need to be defined (not computed) before creating the nodegroup.

Reply:
Should these comments be in the file rather than the PR?

Reply:
I actually added a few lines here https://github.com/atlassian-labs/data-center-terraform/blob/eec1d1f8bc8f1397ea2c003b661eba2a7cd83c3a/modules/AWS/eks/iam.tf#L25